packages/terra-avatar/src/clinical-lowlight-theme/Avatar.module.scss

```diff
@@ -0,0 +1,42 @@
+:local {
+  .clinical-lowlight-theme {
```

Comment:
I did control + f  for `--terra` on `packages/terra-avatar/src/common/Avatar.module.scss` and on this file. I think we are missing two vars in both lowlight & fusion theme